src/main/java/com/frcteam3255/components/swerve/SN_SwerveModule.java

```diff
@@ -80,33 +88,32 @@ public SN_SwerveModule(int moduleNumber, int driveMotorID, int steerMotorID, int
 
 		driveMotor = new TalonFX(driveMotorID, CANBusName);
 		steerMotor = new TalonFX(steerMotorID, CANBusName);
+		driveMotorControllerOpen = new VoltageOut(0);
+		driveMotorControllerClosed = new VelocityDutyCycle(0);
+		steerMotorController = new PositionDutyCycle(0);
 
```

Comment:
are these the correct control types? Open using VoltageOut while closed is using DutyCycle?

Reply:
They were not :D
Changing in the next commit (will upload when it's tested tmr)